main.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"crypto/hmac"
+	"crypto/sha512"
+	"encoding/base64"
+	"encoding/json"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"regexp"
+	"strconv"
+	"strings"
+	"time"
+)
+
+const apiKey = "your api key"
+const privateKey = "your private key in base64"
+const baseUrl = "https://server-host-url"
+
+const apiPath = "/v3/orders"
+
+func main() {
+
+	id := extractOrderId(addOrder())
```

Comment:
for simplicity comment out addOrder and cancelOrder functions and only allow getOpenOrders() function. This is  to void users running undesired trades when running the sample. 

please extract id only upon a successful addOrder (it produces an error when addOrder API returns errors like invalid API key).